gitless/core.py

```diff
@@ -65,14 +65,16 @@ def error_on_none(path):
   return path
 
 
-def init_repository(url=None):
+def init_repository(url=None, exclude=None, only=None):
```

Comment:
put “only” before “exclude” to be consistent with the other functions that take only/exclude